### ISSUE TITLE
Customization in Java / Examples

### ISSUE THREAD

**Issue by wipfli** [onthegomap/planetiler#97]
I would like to provide a customization example in Java. I made a powerlines map here https://github.com/wipfli/powerlines-switzerland and was wondering if we could add it or link it to planetiler?

For users, I think it would be nice to see how to use planetiler as a dependency. Having a separate example project might be useful for this. The powerlines example above has to be adjusted I think.

One thing I found useful when starting with Mapbox GL JS was their collection of examples https://docs.mapbox.com/mapbox-gl-js/example/. These examples have a title, a 3-sentence text, and like 20 lines of code. Maybe we could make something similar where we highlight how to use different features of Planetiler.

**Comment by msbarry** [comment]
Awesome! I tried to make planetiler-examples an example standalone project, but maybe a separate repo would make it easier to grok? I'd be happy to bring in your example to planetiler, or link to a separate repo. Which do you think would be easier for newcomers?

**Comment by wipfli** [comment]
Yes, I think a stand-alone repo is better for people who want to get started...
